crates/ruff_python_formatter/tests/snapshots/black_compatibility@simple_cases__comments2.py.snap

```diff
@@ -382,11 +340,7 @@ short
         # yup
         arg3=True,
     )
-    lcomp = [
-        element  # yup
-        for element in collection  # yup
-        if element is not None  # right
-    ]
+    lcomp = [element for element in collection if element is not None]  # yup  # yup  # right
```

Comment:
This is much closer to Black but still wrong due to the line suffixes not being included when determining the line length:

```python
lcomp = [
    element for element in collection if element is not None  # yup  # yup  # right
]
```